async/format.js

```diff
@@ -29,6 +29,7 @@
  * @function
  */
 module.exports = function (random, alphabet, size) {
+  // TODO
   var mask = (2 << Math.log(alphabet.length - 1) / Math.LN2) - 1
   var step = Math.ceil(1.6 * mask * size / alphabet.length)
```

Comment:
`step` is how much bytes we generate in one round. Hardware random call generator has a big price because we need to wait for entropy collection.

Hardware random generator needs entropy only for first random seed, then it generates next bytes by the algorithm. This is why it is cheaper sometimes to ask for more bytes in the round to reduce calls.

1.6 manually selected constant of how many extra bytes it is better to ask.

Reply:
I didn't change anything because it sounds precise
Done ✅